tests/features/joinup_document/api.feature

```diff
@@ -6,13 +6,14 @@ Feature: Document API
 
   Scenario: Programmatically create a Custom Page
     Given the following collection:
-      | title             | Joinup document name |
-      | owner             | Joinup Derp          |
-      | logo              | logo.png             |
-      | moderation        | yes                  |
-      | closed            | yes                  |
-      | elibrary creation | facilitators         |
+      | uri               | http://joinup.eu/document_api_collection |
+      | title             | Joinup document name                     |
+      | owner             | Joinup Derp                              |
+      | logo              | logo.png                                 |
+      | moderation        | yes                                      |
+      | closed            | yes                                      |
+      | elibrary creation | facilitators                             |
     And document content:
-      | title    | field_document_short_title | body               | collection              |
-      | JD title | Short                      | Dummy description. | http://joinup.eu/jd/jde |
+      | title    | field_document_short_title | body               | og_group_ref                             |
+      | JD title | Short                      | Dummy description. | http://joinup.eu/document_api_collection |
```

Comment:
This is a regression from a BDD standpoint. `og_group_ref` has no meaning for our business owners, but they do understand `collection`. Let's make sure we can still talk about collections and solutions in this step.